### PR TITLE
Combine B2B invoice, payments, and receipt specs

### PR DESCRIPTION
**Pull request by becomevocal**
This enables the specs to be organized under "Invoice Management" in the dev center. Once this is merged and devcenter updates, we can remove the separate invoice, payment, and receipt files.

### CODE REVIEW

**Review by bc-traciporter — COMMENTED** [review]
I resolved all the errors in the file so all checks pass. We may need to do another review for the grammar later.

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 1]
```diff
@@ -0,0 +1,4378 @@
...
+          name: sortBy
+          description: The response sorted by which field
+      description: "Get lines of a receipt, with pagination data"
+    parameters:
+      - schema:
+          type: string
+        name: receiptId
+        in: path
+        required: true
+        description: The unique id of receipt
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:4179]:
```suggestion
        description: The unique ID of the receipt.
```

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 2]
```diff
@@ -0,0 +1,4378 @@
...
+  "/receipts/{receiptId}/lines/{receiptLineId}":
+    parameters:
+      - schema:
+          type: string
+        name: receiptId
+        in: path
+        required: true
+        description: The unique id of receipt
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:4187]:
```suggestion
        description: The unique ID of the receipt.
```

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 3]
```diff
@@ -0,0 +1,4378 @@
...
+      - schema:
+          type: string
+        name: receiptLineId
+        in: path
+        required: true
+        description: The unique id of receipt line
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:4193]:
```suggestion
        description: The unique ID of the receipt line.
```

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 4]
```diff
@@ -0,0 +1,4378 @@
...
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  code:
+                    type: number
+                    description: Response code of success request
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:4357]:
```suggestion
                    description: Response code of a successful request.
```

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 5]
```diff
@@ -0,0 +1,4378 @@
...
+                  data:
+                    type: object
+                    properties:
+                      id:
+                        type: integer
+                        description: The unique id of receipt line
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:4363]:
```suggestion
                        description: The unique ID of the receipt line.
```

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 6]
```diff
@@ -0,0 +1,4378 @@
...
+                  meta:
+                    type: object
+                    properties:
+                      message:
+                        type: string
+                        description: Response message of request
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:4369]:
```suggestion
                        description: Response message of the request
```

**Review by bc-traciporter — COMMENTED** [review]
editorial suggestions

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 7]
```diff
@@ -0,0 +1,4378 @@
...
+              - 2
+            example: 1
+          in: query
+          name: status
+          description: Query by invoice status
+        - schema:
+            type: string
+          in: query
+          name: beginDateAt
+          description: 'Query by the created time period, with the "endDateAt" parameter'
```
Comment (becomevocal) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:1553]:
```suggestion
          description: 'Query by the created timestamp, along with the "endDateAt" parameter'
```

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 8]
```diff
@@ -0,0 +1,4378 @@
...
+                          required:
+                            - paymentId
+                            - invoiceId
+                            - invoiceNumber
+                      customerName:
+                        type: string
+                      allowedOperations:
+                        type: array
+                        description: |-
+                          allowed payment actions
```
Comment (becomevocal) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:2418]:
```suggestion
                          Allowed payment actions:
```

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 9]
```diff
@@ -0,0 +1,4378 @@
...
+                          operation code map as
+                          (0, "Apply to Invoice Balances"),
+                          (1, "Revert Invoice Balances"),
+                          (2, "Pull Updates From Order"),
+                        items:
+                          type: number
+                      allowedStatuses:
+                        type: array
+                        description: |-
+                          allowed payment status
```
Comment (becomevocal) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:2428]:
```suggestion
                          Allowed payment status:
```

**Comment by becomevocal** [comment]
@bc-traciporter I went ahead and pulled your suggestions in. Ready for your approval. You can merge it in as well. Future updates can be made as a separate PR.

File: docs/b2b-edition/specs/api-v3/invoice-management.yaml [hunk 10]
```diff
@@ -0,0 +1,4385 @@
...
+                    properties:
+                      id:
+                        type: integer
+                        description: The unique receipt ID.
+                  meta:
+                    type: object
+                    properties:
+                      message:
+                        type: string
+                        description: Response message of request
```
Comment (bc-traciporter) [inline docs/b2b-edition/specs/api-v3/invoice-management.yaml:3624]:
```suggestion
                        description: Response message of a request
```

**Review by bc-traciporter — APPROVED** [review]

**Pull request merged**